src/main/java/org/broadinstitute/hellbender/engine/filters/CommandLineFilter.java

```diff
@@ -0,0 +1,10 @@
+package org.broadinstitute.hellbender.engine.filters;
+
+import java.io.Serializable;
+
+/**
+ * Interface implemented by command-line accessible read filters for argument validation.
+ */
+public interface CommandLineFilter extends Serializable {
```

Comment:
This name is confusing if you don't know what it means.  Maybe "FilterWithArguments" or something like that?